index.html

```diff
@@ -48,5 +48,162 @@ <h1 class="main-section__introduction">I'm Mariana Glad to see you</h1>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+      <div class="project-items">
+        <img src="images/portfolio1.png" alt="Project photo" />
+        <div class="project-info">
+          <h2>Tonic</h2>
+          <div class="project-details">
+            <span class="project-detail">CANOPY</span>
+            <ul class="project-stack">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+      <div class="project-items">
+        <img src="images/portfolio1.png" alt="Project photo" />
+        <div class="project-info">
+          <h2>Multi-Post Stories</h2>
+          <div class="project-details">
+            <span class="project-detail">CANOPY</span>
+            <ul class="project-stack">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-section">
+      <h2>About<br />Myself</h2>
+
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div>
+        <a href="#"><p>let's connect</p></a>
+
+        <div class="about-socials">
+          <a href=""><img src="images/twitter.png" alt="Twitter logo" /></a>
+          <a href=""><img src="images/twitter.png" alt="Twitter logo" /></a>
+          <a href=""><img src="images/git.png" alt="Git logo" /></a>
+          <a href=""><img src="images/m.png" alt="M logo" /></a>
+          <a href=""><img src="images/nice.png" alt="V logo" /></a>
+        </div>
```

Comment:
- [ ] kindly update your code to make it more semantic, the social icons should be listed inside a list `<ul> & <li>` instead of normal divs, and each social icon should be an `<a>` that contains the link for that icon
example:
```HTML
<ul class='...'>
<li class='...'><a href='...' ><img>...</img></a></li>
<li class='...'><a href='...' ><img>...</img></a></li>
...
<li class='...'><a href='...' ><img>...</img></a></li>
</ul>
```